src/main/scala/com/gu/datalakealerts/Features.scala

```diff
@@ -41,9 +41,18 @@ object Features {
         |and c.component.campaign_code like '%friction%' and c.component.campaign_code like '%subscription_screen%'
         |group by 1""".stripMargin
 
+      // Users with personalised ads enabled should now only see the friction screen on Fri/Sat
+      val peakDays = List(DayOfWeek.FRIDAY, DayOfWeek.SATURDAY)
+      // Users with personalised ads disabled will still see friction screens on other days
+      val offPeakDays = DayOfWeek.values().toList.diff(peakDays)
+
+      val dayOfWeek = LocalDate.now().getDayOfWeek
+
       val minimumImpressionsThreshold = platform match {
-        case Android => 10000
-        case Ios => 40000
+        case Android if peakDays.contains(dayOfWeek) => 45000
+        case Android if offPeakDays.contains(dayOfWeek) => 4000
```

Comment:
I don't understand why there is such a big disparity between the number of off-peak impressions on each platform - I'll look into that separately and adjust these thresholds if necessary.

Reply:
Could it be because on iOS we respect the system limit-ad-tracking flag, so possibly more users receive non-personalised ads?